src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -4,12 +4,29 @@ import _ from 'lodash';
 
 // TO DO: Replace made up properties and content with real versions once finalized.
 export const STATUS_TYPES = {
-  nod: 'nod',
+  // Open Statuses:
+  pendingSoc: 'pending_soc',
   pendingForm9: 'pending_form9',
-  awaitingHearingDate: 'awaiting_hearing_date',
+  pendingCertification: 'pending_certification',
+  pendingCertificationSsoc: 'pending_certification_ssoc',
+  remandSsoc: 'remand_ssoc',
+  pendingHearingScheduling: 'pending_hearing_scheduling',
+  scheduledHearing: 'scheduled_hearing',
   onDocket: 'on_docket',
+  atVso: 'at_vso',
+  decisionInProgress: 'decision_in_progress',
+  bvaDevelopment: 'bva_development',
+  stayed: 'stayed',
+  remand: 'remand',
   bvaDecision: 'bva_decision',
```

Comment:
`bva_decision` is also a closed status. Maybe there are reasons of which I am not aware to not classify it in this way.

Reply:
VSO should be spelled out. Use Veteran Service Organization.

Reply:
Yeah, this segmentation obviously doesn't actually do anything, but thanks for pointing that out. Might as well have it right if I'm going to bother putting in that comment. 🤷‍♂️